system/Debug/Exceptions.php

```diff
@@ -328,6 +336,29 @@ protected function determineCodes(Throwable $exception): array
         return [$statusCode, $exitStatus];
     }
 
+    private function isDeprecationError(int $error): bool
+    {
+        $deprecations = E_DEPRECATED | E_USER_DEPRECATED;
+
+        return ($error & $deprecations) !== 0;
+    }
+
+    private function deprecationErrorHandler(string $message, ?string $file = null, ?int $line = null): bool
```

Comment:
I was expecting this to return some handler object. Can we give this an action name? Like:
```suggestion
    private function handleDeprecationError(string $message, ?string $file = null, ?int $line = null): bool
```